### PR TITLE
Fix/improve prompting, labels, helper for instances

### PR DESCRIPTION
**Pull request by philogicae**
For instances:
- Fixed prompting to skip already defined args (vcpus, memory, etc)
- Add --name
- Fix and move labels to help_strings.py
- Reorganize "instance create --help"
- Default channel to "ALEPHCLOUD-SOLUTIONS" to be displayable on 26 cloud

### CODE REVIEW

File: src/aleph_client/commands/help_strings.py [hunk 1]
```diff
@@ -8,20 +8,33 @@
 CUSTOM_DOMAIN_OWNER_ADDRESS = "Owner address, default current account"
 CUSTOM_DOMAIN_NAME = "Domain name. ex: aleph.im"
 CUSTOM_DOMAIN_ITEM_HASH = "Item hash"
+SKIP_VOLUME = "Skip addition of volume"
```
Comment (olethanh) [inline src/aleph_client/commands/help_strings.py:11]:
```suggestion
SKIP_VOLUME = "Skip prompt to attach more volumes"
```

**Review by olethanh — APPROVED** [review]
Thanks for theses improvements

File: src/aleph_client/commands/instance/__init__.py [hunk 2]
```diff
@@ -212,18 +202,31 @@ def validate_ssh_pubkey_file(file: Union[str, Path]) -> Path:
             if not firmware_message:
                 typer.echo("Confidential Firmware hash does not exist on aleph.im")
                 raise typer.Exit(code=1)
+    if not name:
+        name = validated_prompt("Instance name", lambda x: len(x) < 65)
```
Comment (hoh) [inline src/aleph_client/commands/instance/__init__.py:206]: What about simplifying this ?
```suggestion
    name = name or validated_prompt("Instance name", lambda x: len(x) < 65)
```

**Review by hoh — COMMENTED** [review]
Thanks for fixing this, prompting for values already provided was annoying.

**Comment by philogicae** [comment]
PR ready to be merged after your review

**Pull request merged**